akka-http-core/src/main/resources/reference.conf

```diff
@@ -18,6 +18,17 @@ akka.http {
     # the request, no `Server` header will be rendered at all.
     server-header = akka-http/${akka.http.version}
 
+    # "PREVIEW" features that are not yet fully production ready.
+    # These flags can can change or be removed between patch releases.
+    preview {
+      # ONLY WORKS WITH `bindAndHandleAsync` (currently)
+      # 
+      # If this setting is enabled AND the akka-http2-support is found 
+      # on the classpath the usual Http().bind... method calls will bind 
+      # using HTTP/2. Please note that you must configure HTTPS while doing so.
+      use-http2 = off
```

Comment:
How about `enable-http2`? Otherwise, it sounds a bit as if Http/2 would be used in any case (but it is only used when the client supports it). No strong feelings about it, though.